Libplanet.Net/Consensus/Context.cs

```diff
@@ -468,5 +454,19 @@ private bool HasOneThirdValidators(int round)
         {
             return _messageLog.GetValidatorsCount(round) > TotalValidators / 3;
         }
+
+        private void SaveLastCommit()
+        {
+            if (!(GetBlockCommit() is BlockCommit currentLastCommit))
+            {
+                return;
+            }
+
+            _blockChain.Store.PutLastCommit(currentLastCommit);
+            _logger.Debug(
+                "Saving current height #{Height} and round {Round} LastCommit...",
+                currentLastCommit.Height,
+                currentLastCommit.Round);
+        }
```

Comment:
Now that I look at it, I think direct access to `IStore` from `Context` might not be that great. Since `ConsensusContext` already does the loading, how about moving this to `ConsnesusContext.Commit()`?

Reply:
Sure, and I also forgot to remove the "Save LastCommit with `Dispose()`" unit test. 🤧